jfrog-applications/jfrog-cli/cli-for-jfrog-artifactory/release-lifecycle-management.md

```diff
@@ -330,3 +330,56 @@ Delete the distributions of release bundle named myApp with version 1.0.0 from e
 Delete the distributions of the release bundle associated with project "proj" from the provided edge nodes. Run the command synchronously and skip the confirmation message.
 
 	jf rbd --dist-rules=/path/to/dist-rules.json --project="proj" --quiet --sync myApp 1.0.0
+
+## Exporting Release Bundle Archive
+JFrog Distribution supports distributing your Release Bundles to remote Distribution Edge nodes within an Air Gap environment. This use case is mainly intended for organizations such as financial institutions and military installations that have two or more JFrog Artifactory instances that have no network connection between them.
+
+The procedure below describes how to export a Release Bundle v2 to a .zip file that can be transferred to a different Artifactory instance in an Air Gap environment.
+
+
+|                        |                                                                                                                                                 |
+|------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------|
+| Command-name           | release-bundle-export                                                                                                                           |
+| Abbreviation           | rbe                                                                                                                                             |
+| Command arguments      |                                                                                                                                                 |
+| release bundle name    | Name of the release bundle to distribute.                                                                                                       |
+| release bundle version | Version of the release bundle to distribute.                                                                                                    
+| target pattern         | Specifies the local file system target download path                                                                                                     
```

Comment:
I suggest to put the whole explanation as in the help because users might relay on documentation and not the help command

Reply:
Same comment for `mapping-pattern` and `mapping-target`.

Also, missing `split-count` and `min-split`